tests/unit/testplan/testing/test_pyunit.py

```diff
@@ -0,0 +1,88 @@
+"""Unit tests for the PyUnit test runner."""
+
+import unittest
+import pytest
+
+from testplan.testing import pyunit
+from testplan.testing import filtering
+from testplan.testing import ordering
+from testplan import defaults
+
+PYUNIT_DEFAULT_PARAMS = {
+    "test_filter": filtering.Filter(),
+    "test_sorter": ordering.NoopSorter(),
+    "stdout_style": defaults.STDOUT_STYLE,
+}
+
+
+@pytest.mark.skip("Example PyUnit testsuite")
+class Passing(unittest.TestCase):
+    """
+    Minimal PyUnit testcase with a single trivial test method. For more
+    information about the unittest library, see the [documentation](
+    http://docs.python.org/2/library/unittest.html).
+    """
+
+    def test_asserts(self):
+        """Test with basic assertions."""
+        self.assertTrue(True)
+        self.assertFalse(False)
+
+
+@pytest.mark.skip("Example PyUnit testsuite")
+class Failing(unittest.TestCase):
```

Comment:
We're skipping these so pytest doesn't pick up on the `test_` prefix correct?

Reply:
Right, otherwise PyTest would thing these were testcases it should run and then fail.